### PR TITLE
feat(bucket): read-only option for bucket fields

### PR DESCRIPTION
**Pull request by htuna07**

### CODE REVIEW

File: stacks/api/bucket/src/bucket.schema.resolver.ts [hunk 1]
```diff
@@ -38,8 +38,43 @@ export class BucketSchemaResolver {
...
+  let keywords = [
+    {
+      name: "default",
+      def: {
+        modifying: true,
+        compile: (schema, parentSchema, it) => {
+          return (data, dataPath, parentData) => {
+            console.log(schema, parentSchema);
+
+            console.log(data, dataPath, parentData);
```
Comment (thesayyn) [inline stacks/api/bucket/src/bucket.schema.resolver.ts:59]: You have forgotten to remove these lines.

**Review by thesayyn — CHANGES_REQUESTED** [review]

**Review by thesayyn — APPROVED** [review]

**Comment by thesayyn** [comment]
I want to ask something however. Can I use readonly without default value?

**Comment by htuna07** [comment]
> I want to ask something however. Can I use readonly without default value?

Yes, you can. But the default value will be `undefined` in this case.

**Comment by htuna07** [comment]
> > I want to ask something however. Can I use readonly without default value?
> 
> Yes, you can. But the default value will be `undefined` in this case.

Actually, it won't work without default keyword, sorry.

**Pull request merged**